src/init.cpp

```diff
@@ -119,14 +119,14 @@
 using common::AmountErrMsg;
 using common::InvalidPortErrMsg;
 using common::ResolveErrMsg;
-using kernel::DumpMempool;
-using kernel::LoadMempool;
 using kernel::ValidationCacheSizes;
 
 using node::ApplyArgsManOptions;
 using node::BlockManager;
 using node::CacheSizes;
 using node::CalculateCacheSizes;
+using node::DumpMempool;
+using node::LoadMempool;
```

Comment:
Nit: These should be below `node::DEFAULT_STOPATHEIGHT`.

Reply:
Done.